docs/src/pages/docs/crocks/Async.md

```diff
@@ -966,6 +966,71 @@ resolve(Rejected('Rejected'))
 //=> res: "Was Rejected"
 ```
 
+#### bichain
+
+```haskell
+Async e a ~> ((e -> Async b c), (a -> Async b c)) -> Async b c
+```
+
+Combining a sequential series of transformations that capture disjunction can be
+accomplished with `chain`. Along the same lines, `bichain` allows you to do this
+from both [`Rejected`](#rejected) and [`Resolved`](#resolved). `bichain` expects
+two unary, `Async` returning functions as its arguments. When invoked on 
+a [`Rejected`](#rejected) instance, `bichain` will use
+the [`Rejected`](#rejected) value and can return either a [`Rejected`](#rejected) or
+a [`Resolved`](#resolved). When called on a [`Resolved`](#resolved) instance, it
+will behave exactly as [`chain`](#chain) would.
+
+<!-- eslint-disable no-console -->
+<!-- eslint-disable no-sequences -->
+
+```javascript
```

Comment:
So sorry @dalefrancis88, I was going through the example and I think the meaning of `bichain` may end up getting lost. would it be possible to "focus: this example a bit by:
1. showing just simple "movement" of the instances like we do with Natural Transformation function
2. provide an example showing how to Resolve some errors but Reject others.

Something along the lines of:
```javascript
import bichain from 'crocks/pointfree/bichain'

import Async from 'crocks/Async'
import Maybe from 'crocks/Maybe'

import composeK from 'crocks/helpers/composeK'
import constant from 'crocks/combinators/constant'
import converge from 'crocks/combinators/converge'
import equals from 'crocks/pointfree/equals'
import getProp from 'crocks/Maybe/getProp'
import liftA2 from 'crocks/helpers/liftA2'
import maybeToAsync from 'crocks/Async/maybeToAsync'
import safe from 'crocks/Maybe/safe'

const { Just } = Maybe
const { Rejected, Resolved } = Async

// log :: String -> a -> a
const log = label => x =>
  (console.log(`${label}:`, x), x)

const fork = m =>
  m.fork(log('rej'), log('res'))

fork(
  bichain(Resolved, Rejected, Resolved(42))
)
//=> rej: 42

fork(
  bichain(Resolved, Rejected, Rejected(42))
)
//=> res: 42

// fake401 :: Async Response a
const fake401 = Rejected({
  status: 'Unauthorized',
  statusCode: 401
})

// fake500 :: Async Response a
const fake500 = Rejected({
  status: 'Internal Server Error',
  statusCode: 500
})

// fake200 :: Async e Response
const fake200 = Resolved({
  status: 'OK',
  statusCode: 200
})

// allow401 :: Response -> Async e a
const allow401 = resp => maybeToAsync(
  resp,
  converge(
    liftA2(constant),
    Just,
    composeK(safe(equals(401)), getProp('statusCode'))
  ),
  resp
)

fork(bichain(allow401, Resolved, fake500))
//=> rej: { status: 'Internal Server Error', statusCode: 500 }

fork(bichain(allow401, Resolved, fake401))
//=> res: { status: 'Unauthorized', statusCode: 401 }

fork(bichain(allow401, Resolved, fake200))
//=> res: { status: 'OK', statusCode: 200 }
```